pkg/apis/core/types_controllerregistration.go

```diff
@@ -75,6 +75,8 @@ type ControllerResource struct {
 	//  Delete: "BeforeKubeAPIServer"
 	//  Migrate: "BeforeKubeAPIServer"
 	Lifecycle *ControllerResourceLifecycle
+	// WorkerlessSupported specifies whether this ControllerResource supports Workerless Shoot clusters.
```

Comment:
```suggestion
	// WorkerlessSupported specifies whether this ControllerResource supports Workerless Shoot clusters.
	// This field is only relevant when kind is "Extension".
```

same for versioned APIs